projects/packages/videopress/src/client/block-editor/blocks/video/types.ts

```diff
@@ -98,8 +98,8 @@ export type DetailsPanelProps = VideoControlProps & {
 };
 
 export type VideoPreview = {
-	html?: string;
-	scripts: Array< string >;
-	width?: number;
-	height?: number;
+	html: string;
+	scripts?: Array< string >;
```

Comment:
I'm not sure if the `scripts` key is expected here.

Reply:
I see that `scripts` is not part of the API response but it looks like it is [attached to the preview](https://github.com/Automattic/jetpack/blob/f364a7247f7c50a2d7d38e176a0c382e88407e16/projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx#L162) at some point, I suspect from the `select( coreStore ).getEmbedPreview` flow.

Reply:
I think we (I) added the `scripts` prop to the Player component wrongly. I suggest diggin into it in a follow-up.

Reply:
I actually haven't dug too far into the `preview` destructing in `edit.ts`. Is it possible that the scripts prop is not on the preview that it fetched from the store ?

Reply:
So I just logged the preview object and looks like `scripts` is not there hence the[ destructing of the preview ](https://github.com/Automattic/jetpack/blob/f364a7247f7c50a2d7d38e176a0c382e88407e16/projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx#L563) in edit is not useful.